src/backend/api/meals.js

```diff
@@ -1,12 +1,161 @@
 const express = require("express");
 const router = express.Router();
 const knex = require("../database");
+const { parse } = require("path");
+
+//NOTE: Everything works except for the availableReservations param route when combnining with other params.
+//SQL strict mode won't let groupping in this case - help needed :)
 
 router.get("/", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    const maxPrice = request.query.maxPrice;
+    const availableReservations = request.query.availableReservations;
+    const title = request.query.title;
+    const createdAfter = request.query.createdAfter;
+    const limit = request.query.limit;
```

Comment:
You can also do like `const { maxPrice, limit, ...} = request.query` then request query will be an object with all the queries.

Reply:
Hi :) Sorry for the slow reply and thanks a lot for your feedback 🤓 That's a really nice point!